src/streamsync/__init__.py

```diff
@@ -38,3 +39,10 @@ def init_state(state_dict: Dict[str, Any]):
     initial_state.user_state.state = {}
     initial_state.user_state.ingest(state_dict)
     return initial_state
+
+
+def init_ui():
```

Comment:
I would advocate to add typing information using Union of Generator (for mypy and vscode) and ContextManager (for pycharm)